composer.json

```diff
@@ -19,7 +19,9 @@
     ],
     "require-dev": {
         "cakephp/cakephp": "~3.0",
-        "friendsofcake/cakephp-test-utilities": "dev-master"
+        "friendsofcake/cakephp-test-utilities": "dev-master",
+        "cakephp/cakephp-codesniffer": "2.*",
+        "phpunit/phpunit": "^5.5"
```

Comment:
Please don't change the composer file. You can install codesniffer globally locally. Also phpunit 5.5 requires PHP 5.6+ and we need to test on 5.4 and. So just undo changes to composer.json